src/libopensc/pkcs15-pubkey.c

```diff
@@ -1617,9 +1617,9 @@ sc_pkcs15_convert_pubkey(struct sc_pkcs15_pubkey *pkcs15_key, void *evp_key)
 			return SC_ERROR_INTERNAL;
 		X = BN_new();
 		Y = BN_new();
-		if (X && Y && EC_KEY_get0_group(eckey))
-			r = EC_POINT_get_affine_coordinates_GFp(EC_KEY_get0_group(eckey),
-					point, X, Y, NULL);
+//		if (X && Y && EC_KEY_get0_group(eckey))
+//			r = EC_POINT_get_affine_coordinates_GFp(EC_KEY_get0_group(eckey),
+//					point, X, Y, NULL);
```

Comment:
Why did you remove this code?

Reply:
Unfortunately the italian way of providing a signature device from 2009 onwards was to lock-in to proprietary library that comes along with the device, based on Secure Messaging symmetric keys embedded in the PKCS11 library.
The cards of which I ask the removal of the support have been considered unusable starting from 1/7/2011 (unfortunately I have not yet found a document in English).
For the lines commented on pkcs15-pubkey.c it's my mistake, now I put it back as it was.

Reply:
So am I right saying that today
- there aren't any cards issued that were compatible with OpenSC and
- there aren't any cards in the field that were compatible with OpenSC and still have a valid certificate?

And I assume that the new proprietary library also has support for old cards so that users are still able to decrypt their old mails. Correct?

Reply:
> * there aren't any cards issued that were compatible with OpenSC and
> * there aren't any cards in the field that were compatible with OpenSC and still have a valid certificate?

Yes
 
> And I assume that the new proprietary library also has support for old cards so that users are still able to decrypt their old mails. Correct?

This risk does not exist because the certificate inserted in those cards had only the use not-repudiation, so no sign, encrypt or decrypt